test/JDBC/expected/BABEL-2805.out

```diff
@@ -59,3 +59,40 @@ int#!#varchar#!#varchar
 
 DROP TABLE t1;
 GO
+
+CREATE FUNCTION fn_GetMaxDate
```

Comment:
Let's move this test case to the upgrade test and use our function name convention, e.g., `babel_2805_vu_f1`.

Reply:
Done